mapswipe_workers/python_scripts/results_to_mapping_sessions_results.py

```diff
@@ -0,0 +1,66 @@
+import sys
+
+import pandas as pd
+
+from mapswipe_workers import auth
+from mapswipe_workers.definitions import logger
+
+
+def copy_results_batch(first_timestamp, last_timestamp):
+    logger.info(
+        f"Start process for : ms.start_time>={first_timestamp} "
+        f"and ms.start_time<{last_timestamp}"
+    )
+    p_con = auth.postgresDB()
+    query = """
+        -- create table with results for given time span
+        drop table if exists results_batch;
+        create table results_batch as
+        select r.*
```

Comment:
```suggestion
        select ms.mapping_session_id, r.task_id, r."result"
```
if you add it here, I think you can skip the second join below (line 35-37) as you'd have all the info ready in your temp table, no?